content/en/platform/corda/5.0/deploying-operating/observability/metrics.md

```diff
@@ -230,3 +235,54 @@ the maximum value, and the cumulative sum of values, respectively.
 In addition, the Corda metrics endpoint also includes Caffeine cache metrics (`corda_cache_*`),
 Kafka producer and consumer client metrics (`corda_kafka_*`), JVM metrics (`jvm_*`), process metrics (`process_*`),
 and system metrics (`system_*`) provided by the corresponding Micrometer bindings.
+
+#### Database Worker
+
+The database worker is the sole worker with access to the database, apart from the crypto worker, which has its
+own dedicated database. As a result, the activities carried out within the database worker pertain exclusively to the database.
```

Comment:
Nitpicky: Maybe I 'd put the reference to crypto worker inside parentheses to demote its presence here. This is just a mention to be more accurate, it could be omitted.

Reply:
Added the parenthesis.